### PR TITLE
FindFreeQuadObj

### PR DESCRIPTION
**Pull request by sozud**

### CODE REVIEW

File: include/common.h [hunk 1]
```diff
@@ -417,6 +417,12 @@ struct GameInfo {
     u8 unkD;
 };
 
+typedef struct {
+    s8 first_byte;
+    u8 padding[95];
+} QuadObj;
```
Comment (joshlory) [inline include/common.h:423]: I think this overlaps with an existing type defined here:

https://github.com/sozud/mmx4/blob/766bd9ca0d64f1c289b3419639d25ce1b0b2830f/include/common.h#L296-L299

File: src/main/323C.c [hunk 2]
```diff
@@ -1434,7 +1436,20 @@ INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_8002AE90);
 
 INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_8002AF4C);
 
-INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_8002B008);
+struct QuadObj* FindFreeQuadObj()
```
Comment (joshlory) [inline src/main/323C.c:1439]: Nit: why not `find_free_quad_obj` to match the other `find` funcs? Should we update those to match?

File: include/common.h (hunk 1 above)
Comment (sozud) [reply include/common.h:423]: You're right, I'll do a followup

File: src/main/323C.c (hunk 2 above)
Comment (sozud) [reply src/main/323C.c:1439]: Yeah I've been indecisive on naming style and we've ended up with a mix. I suppose we ought to pick something. I'm contemplating the rust style? https://doc.rust-lang.org/1.0.0/style/style/naming/README.html

Basically in C terms  `snake_case` for functions, `CamelCase` for types, `SCREAMING_SNAKE_CASE` for constants, `snake_case` for variables?

Open to suggestions

Comment (joshlory) [reply src/main/323C.c:1439]: No strong preference, happy to align to whatever you choose!

Comment (sozud) [reply src/main/323C.c:1439]: Ok let's go with that then, I don't think your PR needs any updating in that case, I'll adjust my name when combining the redundant QuadObjs

**Pull request merged**